edgexfoundry/edgex-compose
# build: Update device-coap & device-bacnet to 3.0.1 patch version

fix: #411 







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions

make run ds-coap ds-bacnet-ip ds-bacnet-mstp
Verify that the logs of device-coap contain the following message:
```
level=INFO ts=2023-08-29T05:28:47Z app=device-coap msg="Starting device-coap device service, version 3.0.1"
level=INFO ts=2023-08-29T05:28:47Z app=device-coap msg="EdgeX device SDK for C, version 3.0.2"
```
Verify that the logs of device-bacnet-ip contain the following message:
```
level=INFO ts=2023-08-29T05:28:47Z app=device-bacnet-ip msg="Starting device-bacnet-ip device service, version 3.0.1"
level=INFO ts=2023-08-29T05:28:47Z app=device-bacnet-ip msg="EdgeX device SDK for C, version 3.0.2"
```
Verify that the logs of device-bacnet-mstp contain the following message:
```
level=INFO ts=2023-08-29T05:34:05Z app=device-bacnet-mstp msg="Starting device-bacnet-mstp device service, version 3.0.1"
level=INFO ts=2023-08-29T05:34:05Z app=device-bacnet-mstp msg="EdgeX device SDK for C, version 3.0.2"
```
